network-api/networkapi/wagtailpages/pagemodels/mixin/foundation_banner_inheritance.py

```diff
@@ -0,0 +1,20 @@
+# A mixin that imparts banner inheritance for whatever model it gets tacked onto
+class FoundationBannerInheritanceMixin():
+
+    def get_banner(self):
+        if self.banner is None:
+            ancestors = self.get_ancestors()
+            root = ancestors[1].specific
```

Comment:
wondering why we we `ancestors[1]` instead of `ancestors[0]`?

Reply:
ancestors[0] is a wagtail level Page literally called `Root`, rather than our own page.